Library/Homebrew/os/mac.rb

```diff
@@ -44,14 +44,14 @@ def latest_stable_version
       # TODO: bump version when new macOS is released and also update
       # references in docs/Installation.md and
```

Comment:
Have you checked this?

Reply:
Looks like Installation.md does need updating too...